security/cmd/istio_ca/main.go

```diff
@@ -244,14 +235,13 @@ func init() {
 
 func main() {
 	if err := rootCmd.Execute(); err != nil {
-		log.Errora(err)
-		os.Exit(-1)
+		log.Fatala(err)
```

Comment:
In Mixer, command-line parsing errors are output to stderr. Once command-line parsing is done and we're operating as a server, then all output goes to the log. This gives a better experience when interacting with the component live.